ADMVEDU/Network/NetworkError.swift

```diff
@@ -21,7 +17,18 @@ extension NetworkError {
 }
 
 extension NetworkError: LocalizedError {
-	public var errorDescription: String? {
-		return ErrorLocalization.networkErrorLocalization(error: self)
+	var localized: String? {
+		switch self {
+		case .invalidData:
+			return localizedError(error: "The provided data is not valid.", comment: "")
+		case .notFound:
+			return localizedError(error: "The specified item could not be found.", comment: "Resource Not Found")
+		case .unexpected:
+			return localizedError(error: "An unexpected error occurred.", comment: "Unexpected Error")
+		}
+	}
+
+	private func localizedError(error: String, comment: String) -> String {
```

Comment:
We should localise not only error description. It can be any string, so there should be localized property of String (via extension) without parameter as a string is a key